tests/test_cases/issue_253/Makefile

```diff
@@ -44,21 +44,21 @@ $(COCOTB_RESULTS_FILE):
 	@echo "Skipping"
 
 notset_top_level: $(SIM_BUILD)/sim.vvp $(CUSTOM_SIM_DEPS) $(COCOTB_LIBS) $(COCOTB_VPI_LIB)
-	PYTHONPATH=$(LIB_DIR):$(COCOTB_PY_DIR):$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
+	PYTHONPATH=$(COCOTB_PY_DIR)/cocotb:$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
```

Comment:
What's going on here? Does `COCOTB_PY_DIR` need to be here at all?

Reply:
xref #253

Reply:
It looks like not anymore. 

We do not need `PYTHONPATH` at all if we set `PYTHONHOME` like in https://github.com/cocotb/cocotb/pull/1462

Reply:
Let's remove `COCOTB_PY_DIR` from this file in this PR, since it's not there in any other test anyway.

Reply:
Or at least, not add `/cocotb` which doesn't make sense to me.

Reply:
```suggestion
	PYTHONPATH=$(COCOTB_PY_DIR):$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
```
or
```suggestion
	PYTHONPATH=$(PWD):$(NEW_PYTHONPATH) $(LIB_LOAD) MODULE=$(MODULE) \
```